source/api/annex/notes/design_patterns.md

```diff
@@ -112,11 +114,64 @@ IIIF specifications define the functionality that can be expected to work and ho
 
 The intent of adopting this pattern is to enable experimentation by implementers, thereby encouraging the early adoption and validation of new (minor) versions.
 
+## 3. JSON-LD Design Patterns
+
+The representation of the information in JSON must be:
+
+  * Valid JSON and JSON-LD, following the restrictions of both specifications ;
+  * Understandable by developers without special training or documentation ;
+  * Usable in a variety of application environments to ensure the widest adoption.
+
+These design patterns have been selected in order to maximize these features in the resulting APIs.
+
+### 3.1. Use Native Data Types
+
+Native data types in JSON are used to ensure that applications do not have to do type coercion. If the value should be an integer in the model, then the JSON representation is as a JSON number.  Similarly, if the value is an ordered list of resources, then a JSON array will be used, mapping to an rdf:List.
+
+### 3.2. Avoid Special Characters
+
+Special characters are avoided in the names of classes and properties. Any character other than those in the range 0-9, a-z, A-Z and the underscore character are removed in the mapping to JSON-LD, if present in the name of a class or property. This is to ensure that applications developed in object oriented programming languages can treat these values as class and property names internally.
+s
+JSON-LD defines two special keys in the data: `@id` (the URI of the resource) and `@type` (the class of the resource).  These are aliased to `id` and `type` respectively to avoid the use of the `@` symbol.
+
+### 3.3. Use Consistent Naming Conventions
+
+Even within the limited range of characters available, there are additional design patterns to make it easier to remember.
```

Comment:
* Delete "Even" or perhaps reorder to "Additional design patterns are used make names easier to remember while keeping to the limited range of characters."?
 * s/it/names/ ?